src/js/components/Ballot/CandidateModal.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component, PropTypes } from "react";
```

Comment:
Hi Steve, I remembered what triggers CandidateModal and MeasureModal -- it comes from clicking on the position bar (once you have followed any organizations that have opinions on either the candidate or modal). I will test if you don't beat me to it.